templates/repo/issue/view_content/pull.tmpl

```diff
@@ -342,6 +342,8 @@
 									'baseLink': {{.Link}},
 									'textCancel': {{$.i18n.Tr "cancel"}},
 									'textDeleteBranch': {{$.i18n.Tr "repo.branch.delete" .HeadTarget}},
+									'textTagName': {{.i18n.Tr "repo.release.tag_name"}},
```

Comment:
I think we should use `{{.locale.Tr ` now?

Reply:
yes, fixed now after merging main again